addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -139,9 +143,11 @@ DuplicatePageDialog.template = xml`
 </WebsiteDialog>
 `;
 DuplicatePageDialog.props = {
+    duplicate: {type: Function, optional: true},
```

Comment:
Adding `duplicate` as a new props while:
1. There is already a `onDuplicate` props
2. The component has a `duplicate()` method (called by its template on click)

Seems hard to figure to me. Correct me if I am wrong but isn't the goal of this new props to actually override / force the `duplicate()` method to something else (and ignore the original one)?
I guess it kinda makes sense to name it `duplicate` but wouldn't it be clearer with something like `overrideDuplicate`?

You don't have to especially agree with my reasoning here, I am genuinely wondering what you think about it